config/prod/iatlas-db.yaml

```diff
@@ -0,0 +1,18 @@
+template_path: aurora-postgresql-db.yaml
+stack_name: iatlasdb-prod
+dependencies:
+  - common/iatlasvpc.yaml
+  - prod/iatlas-kms-prod.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  VpcName: iatlasvpc
+  KmsKeyName: "/alias/iatlas-kms-prod/KmsKey"
+  MasterUsername: !ssm /iatlas/prod/AuroraUsername
+  MasterUserPassword: !ssm /iatlas/prod/AuroraPassword
+  DBInstanceClass: "db.t3.large"
```

Comment:
[Optional] maybe you can set defaults for these DB configs so you won't need to put them here?

Reply:
I think they should be defined here in order to be explicit, so it can be easy to find if we want to override it later.

Reply:
ok